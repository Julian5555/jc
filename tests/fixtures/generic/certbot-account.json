{"account":{"server":"https://acme-staging-v02.api.letsencrypt.org/directory","url":"https://acme-staging-v02.api.letsencrypt.org/acme/acct/12345679","email":"some@example.com"}}
