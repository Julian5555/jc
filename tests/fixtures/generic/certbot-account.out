Saving debug log to /var/log/letsencrypt/letsencrypt.log
Account details for server https://acme-staging-v02.api.letsencrypt.org/directory:
  Account URL: https://acme-staging-v02.api.letsencrypt.org/acme/acct/12345679
  Email contact: some@example.com

